src/Utils/Helpers.ts

```diff
@@ -135,3 +135,10 @@ export async function saveDirtyDocuments(filepath: string): Promise<boolean> {
     return false;
   }
 }
+
+// TODO Manager target file ext in ONE Explorer
+export function isOneExplorerTargetFile(uri: vscode.Uri): boolean {
+  const path = uri.path;
```

Comment:
Currently, `CircleGraph` has similar bug like following.

```suggestion
  const path = uri.fsPath;
```

Is `uri.path` is correct? :)

Reply:
I'll apply it! Thank you!